06_modules_lab/03.py

```diff
@@ -1,11 +1,20 @@
-""" Write a program that searches current working directory
-for files larger than 1MB. Every time you find such a file print
-its name to the user.
-
-- When the program finds a large file. It should ask the user
-  a message asking if she wants to delete it, and delete the
-  file if requested
-
-- Take threshold and path as command line arguments
-"""
+#!/usr/bin/python
+import sys
+import os 
+MB_IN_B= 1048576
+file_to_delete = []
+dir_location = sys.argv[1]
+size_wanted = int(sys.argv[2])
+print dir_location
+print size_wanted
 
+for root, subDirs , files in os.walk(dir_location):
+		for subDir in subDirs:
+			for f in files:
+				if os.path.getsize(os.path.join(root,f)) > size_wanted*MB_IN_B: 
+					file_to_delete.append(os.path.join(root,f))
+if raw_input('would yo like delete all files that are bigger then %s mb  ' % size_wanted ) == 'yes':
```

Comment:
This works however the question was to ask the user before deleting each file, while you ask just once for all files. Nonetheless good solution